deposit/src/main/java/edu/unc/lib/deposit/normalize/BioMedToN3BagJob.java

```diff
@@ -15,14 +15,7 @@
  */
 package edu.unc.lib.deposit.normalize;
 
-import static edu.unc.lib.deposit.work.DepositGraphUtils.cdrprop;
-import static edu.unc.lib.deposit.work.DepositGraphUtils.dprop;
-import static edu.unc.lib.dl.util.ContentModelHelper.CDRProperty.hasSourceMetadataProfile;
-import static edu.unc.lib.dl.util.ContentModelHelper.CDRProperty.sourceMetadata;
 import static edu.unc.lib.dl.util.ContentModelHelper.Datastream.MD_SOURCE;
```

Comment:
One last remaining ContentModelHelper lurking here.  I think you are handling a similar case to this in the Proquest job?